import_dataset/schema_less_guide.md

```diff
@@ -0,0 +1,305 @@
+
+
+## <a name="dataprocessing"></a>Data Processing
+
+After the schema-less design has been decided upon above, it’s time for the data import to begin! The data processing and cleaning process involves the follow stages:
+
+
+
+1. **Data cleaning**: Once the data is collected, it needs to be cleaned and prepared for import. This includes tasks such as removing duplicate data, correcting errors, and standardizing the format of the data. Most of these high level issues should have already been identified and addressed in the Dataset Type section of the template; now, make sure to follow through and actually clean the data accordingly.
+
+2. **Data transformation**: After cleaning, the data is transformed to fit the DataCommons schema, which is based on the schema.org vocabulary. This includes tasks such as mapping the data to the appropriate entities and properties (see Tips for Data Cleaning and Transformation below). Example scripts have been checked into the [data repository](https://github.com/datacommonsorg/data/tree/master/scripts).
+
+3. **Data loading**: After the data is transformed, it is loaded into the DataCommons.org GitHub, where it is made available for use by other users. If the data source is new (e.g. there are not yet other datasets from this source in the [scripts](https://github.com/datacommonsorg/data/tree/master/scripts) folder), create a new folder with the Source name.
+
+4. **Data validation**: After the data is loaded, the data quality assurance process is done which includes tasks such as checking for errors, inconsistencies, and missing data. Data commons has a “dc-import” tool to help with this validation.
+
+5. **Data review**: After you validate your new dataset structure, submit your PR for review by the DC team for final approval.
+
+
+Throughout the process, the data import and cleaning may be aided by scripts and tools that are available in the GitHub repository of DataCommons.org. These scripts are designed to automate many of the tasks involved in the process, making it more efficient and accurate.
+
+A clear overview of the data cleaning and transformation process can be found [here](https://github.com/datacommonsorg/data/blob/master/docs/life_of_a_dataset.md). 
+
+
+### <a name="tipsfordatacleaningandtransformation"></a>Tips for Data Cleaning and Transformation
+
+Your data may be cleaned and transformed in the following ways; please identify in the import template in which ways your data was transformed:
+
+
+* Filtering: Invalid data points, outliers, duplicates
+* Data transformations such as mapping data attributes to schema, unit conversions, scaling, data types for different fields, such as string vs int vs float
+* consider ways to regenerate the same DCID even after more schema properties are added to the SV.
+* Computations: Any derived stats such as area of geoJsons, etc
+* Place resolution
+* Entity resolution
+* libraries/frameworks used : geo, html, etc
+* Dependencies: data, scripts, libraries
```

Comment:
did some indentations get lost here?

Reply:
the casing was a typo, thanks for catching it!

Reply:
Hi Carolyn, you're right, it doesn't seem relevant here and feels out of place. I removed it since there is already a standalone section on dcids and we're doing schemaless here anyhow!